packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -727,35 +726,7 @@ function shc(content: unknown): SanitizedHtmlContent {
  * https://github.com/vuejs/core/blob/e790e1bdd7df7be39e14780529db86e4da47a3db/packages/shared/src/normalizeProp.ts#L63-L82
  */
 function ncls(value: unknown): string | undefined {
-    if (isUndefined(value) || isNull(value)) {
-        // Returning undefined here improves initial render cost, because the old vnode's class will be considered
-        // undefined in the `patchClassAttribute` routine, so `oldClass === newClass` will be true so we return early
-        return undefined;
-    }
-
-    let res = '';
-
-    if (isString(value)) {
-        res = value;
-    } else if (isArray(value)) {
-        for (let i = 0; i < value.length; i++) {
-            const normalized = ncls(value[i]);
-            if (normalized) {
-                res += normalized + ' ';
-            }
-        }
-    } else if (isObject(value) && !isNull(value)) {
-        // Iterate own enumerable keys of the object
-        const keys = ObjectKeys(value);
-        for (let i = 0; i < keys.length; i += 1) {
-            const key = keys[i];
-            if ((value as Record<string, unknown>)[key]) {
-                res += key + ' ';
-            }
-        }
-    }
-
-    return StringTrim.call(res);
+    return normalizeClass(value);
```

Comment:
I think we can simply do `const ncls = normalizeClass` here. And then the above function comment is unnecessary.